cpp/src/io/parquet/bloom_filter_reader.cu

```diff
@@ -655,7 +646,7 @@ std::optional<std::vector<std::vector<size_type>>> aggregate_reader_metadata::ap
     sources, input_row_group_indices, equality_col_schemas, total_row_groups, stream, aligned_mr);
 
   // No bloom filter buffers, return the original row group indices
-  if (bloom_filter_data.empty()) { return std::nullopt; }
+  if (bloom_filter_data.empty()) { return {std::nullopt, false}; }
```

Comment:
Is the comment (that we "return the original row group indices") in conflict with returning `std::nullopt`?

Reply:
Ah yes you are right. Let me update the comment.